### DISCUSSION TITLE
Nftables does not enable between reboots

### DISCUSSION THREAD

**Discussion by SquigglyDog** [wirefalls/geo-nft#29] [Q&A] [answered]
Geo-nft works as I had intended but does not work after I reboot
`systemctl status nftables
○ nftables.service - nftables
     Loaded: loaded (/lib/systemd/system/nftables.service; disabled; vendor pre>
     Active: inactive (dead)
       Docs: man:nft(8)
             http://wiki.nftables.org

`
I have set geo-nft.conf as per methods

> Set the enable_refill setting to yes.

enable_refill=yes
Confirm that restart_nftables=no in the configuration file (the default), since that setting is mutually exclusive with the enable_refill setting. After editing, save the file and exit the text editor. The refill-sets.nft script will now run automatically after a database update.

Not sure if I missed a step bbut these commands appear to have corrected the problem of nftables not being enabled on reboot
`$ sudo systemctl enable nftables
$ sudo systemctl start nftables
$ sudo systemctl status nftables`

**Comment by wirefalls** [comment]
One thing that can keep nftables from loading is an include statement in the `nftables.conf` file that points to a file that doesn't exist. The last line in your `nftables.conf` file should look like this:
```
include "/etc/nftables/geo-nft/*refill-sets.nft"
```
The asterisk just before the filename allows nftables to load even if the file can't be found. If there's an include statement anywhere in the `nftables.conf` file without the asterisk and the file it points to can't be found, then nftables may fail to load.

You can also comment out the line above when testing to determine if the problem with nftables not loading is caused by geo-nft's configuration or something else.

I'm not sure that I understand your last comment correctly or not. Is your setup working now or are you still having issues with nftables not loading after a reboot?

Thanks

**Comment by SquigglyDog** [comment, marked as answer]
seems to work after reboot. No errors when I run `sudo geo-nft`
Thanks
